src/game/client/components/render_layer.h

```diff
@@ -52,6 +52,7 @@ class CRenderLayer : public CComponentInterfaces
 	virtual bool DoRender(const CRenderLayerParams &Params) const = 0;
 	virtual bool IsValid() const { return true; }
 	virtual bool IsGroup() const { return false; }
+	virtual void Unload() {}
```

Comment:
Can this be pure virtual?

```suggestion
	virtual void Unload() = 0;
```

Reply:
AFAICT every render layer class is either group, tile or quads. Only `CRenderLayerGroup` doesn't override `Unload`, so you could also override it there with an empty implementation, like for `Init`.

Reply:
I guess I'll overwrite it anyway and properly do this for some physics layers